### PR TITLE
Hungarian translation

### PR DESCRIPTION
**Pull request by Pikarkapitany**
I added a Hungarian translation for the messages and the GUI,

### CODE REVIEW

**Review by dbear496 — APPROVED** [review]
I gave this a once-over with Google Translate, and it looks pretty good. I just noticed a few things you may want to double-check. Also, I think you need to update your messages.pot file to get updated strings.

File: data/locale/gui/hu.po [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+#: data/gui/mainmenu.xml data/gui/loadgame.xml
+msgid "Load"
+msgstr "Betöltés"
+
+#: data/gui/loadgame.xml
+msgid "Load Game"
+msgstr "Játék betöltése"
+
+#: data/gui/pbar2nd.xml
+msgid "MWh"
```
[the remark this answers is not in the document]
Comment (Pikarkapitany) [reply data/locale/gui/hu.po]: corrected

File: data/locale/gui/hu.po [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+msgid "Map Size"
+msgstr "Térkép méret"
+
+#: data/gui/millup.xml
+msgid "Mills available"
+msgstr "Malom elérhető"
+
+#: data/gui/mod_wind_up.xml
+msgid "Modern Windmills available"
+msgstr "Modern malom elérhető"
```
[the remark this answers is not in the document]
Comment (Pikarkapitany) [reply data/locale/gui/hu.po]: corrected

File: data/locale/gui/hu.po [hunk 3]
```diff
@@ -0,0 +1,809 @@
...
+msgid "You have a budget deficit. The legislative council will not allow you to build universities on credit."
+msgstr "Költségvetési hiányod van. A törvényhozó tanács nem engedi, hogy hitelből egyetemeket építs."
+
+#: data/gui/launch-evac.xml
+msgid "You have evacuated 1000 people. You might want to reduce the size of your economy to reflect the reduced population. "
+msgstr "1000 embert evakuáltál. A gazdaság méretét a népességcsökkenéshez kell igazítanod."
+
+#: data/gui/blacksmithup.xml
+msgid "You have reached the tech level which enables you to build blacksmiths. Blacksmiths use jobs and steel to make goods."
+msgstr "Elérted azt a technikai szintet, amely lehetővé teszi, hogy kovácsműhelyeket építs. A kovácsműhelyek munkaerőt és acélt használnak fel."
```
[the remark this answers is not in the document]
Comment (Pikarkapitany) [reply data/locale/gui/hu.po]: corrected

File: data/locale/gui/hu.po [hunk 4]
```diff
@@ -0,0 +1,809 @@
...
+msgid "You have reached the tech level which enables you to build modern, more efficient, windmills. "
+msgstr "Elérted azt a technikai szintet, amely lehetővé teszi, hogy modern hatékony szélmalmokat építs."
+
+#: data/gui/parkup.xml
+msgid "You have reached the tech level which enables you to build parks. Parks are used to absorb pollution. "
+msgstr "Elérted azt a technikai szintet, amely lehetővé teszi, hogy parkokat építs. A parkokat a légszennyezettség csökkentésére használhatod."
+
+#: data/gui/potteryup.xml
+msgid "You have reached the tech level which enables you to build potteries. Potteries use jobs and ore to make goods. (You need ore mines to get ore) "
+msgstr "Elérted azt a technikai szintet, amely lehetővé teszi, hogy fazekasműhelyeket építs. A fazekasműhelyek munkaerőt és ércet használnak az áruk előállításához. Egy ércbányára is szükséged van, hogy építhess egyet."
```
[the remark this answers is not in the document]
Comment (Pikarkapitany) [reply data/locale/gui/hu.po]: Corrected

**Comment by Pikarkapitany** [comment]
Regenerated messages.pot files, findings corrected

**Comment by dbear496** [comment]
I will re-open this because I think this may have been closed by accident.

**Comment by dbear496** [comment]
Turns out I need to open a separate PR because Picard deleted is fork.

**Pull request closed without merging**